include/openPMD/IO/JSON/JSONIOHandlerImpl.hpp

```diff
@@ -288,7 +288,7 @@ namespace openPMD
         std::shared_ptr< FILEHANDLE > getFilehandle(
             File,
             AccessType accessType
-        ); //, AccessType accessType=this->m_handler->accessType);
+        ); //, AccessType accessTypeFrontend=this->m_handler->accessTypeFrontend);
```

Comment:
@franzpoeschel Is this code safe to remove?

Reply:
@franzpoeschel ping. can we I merge this?

Reply:
@franzpoeschel I will merge this for now, but you can post here and we can do follow-up PRs to `dev` is something is off :)